types/activex-excel/index.d.ts

```diff
@@ -3849,12 +4196,77 @@ declare namespace Excel {
         readonly ChartArea: ChartArea;
         ChartObjects(Index?: SafeArray<string | number>): ChartObjects;
         ChartObjects(Index: string | number): ChartObject;
-        ChartStyle: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | 20 | 21 | 22 | 23 | 24 | 25 | 26 | 27 | 28 | 29 | 30 | 31 | 32 | 33 | 34 | 35 | 36 | 37 | 38 | 39 | 40 | 41 | 42 | 43 | 44 | 45 | 46 | 47 | 48;
+        ChartStyle:
+            | 1
+            | 2
+            | 3
+            | 4
+            | 5
+            | 6
+            | 7
+            | 8
+            | 9
+            | 10
+            | 11
+            | 12
+            | 13
+            | 14
+            | 15
+            | 16
+            | 17
+            | 18
+            | 19
+            | 20
+            | 21
+            | 22
+            | 23
+            | 24
+            | 25
+            | 26
+            | 27
+            | 28
+            | 29
+            | 30
+            | 31
+            | 32
+            | 33
+            | 34
+            | 35
+            | 36
+            | 37
+            | 38
+            | 39
+            | 40
+            | 41
+            | 42
+            | 43
+            | 44
+            | 45
+            | 46
+            | 47
+            | 48;
```

Comment:
Perhaps types defined as a set of numeric constants should also be on a single line?

Reply:
Not much that can be done here, unless a `// prettier-ignore` is added to the definition. This is prettier's chosen format because the line got over its max `printWidth`.

Reply:
Is that an acceptable strategy -- to add `// prettier-ignore`

Reply:
I'm not sure of what the team thinks; but personally, and at work, `// prettier-ignore` is _never_ used unless it's to work around a prettier bug where it would create invalid syntax or unstable formatting (i.e. when the formatting would change every time prettier runs).

So, in this case, I think we'd just have to deal with the newlines.

Reply:
Same for us. In the long run, we find that letting go of some of these formatting preferences makes it easier/more efficient for people to contribute as all thinking about formatting is taken out of the equation and leads to a consistent code-base that reduces cognitive load on maintainers.